### ISSUE TITLE
Deserialize error

### ISSUE THREAD

**Issue by nstrom1** [tymor22/tm-vec#29]
Hi, I am trying to reproduce the example where bacteriocins are annotated but am running into this "deserialize" error. I specified 'cpu' in the command, but I get the same error if I specify 'gpu.'

```
tmvec-build-database \
    --input-fasta bagel.fa \
    --tm-vec-model tm_vec_cath_model.ckpt \
    --tm-vec-config-path tm_vec_cath_model_params.json \
    --device 'cpu' \
    --output bagel_database
```

RuntimeError: Attempting to deserialize object on a CUDA device but torch.cuda.is_available() is False. If you are running on a CPU-only machine, please use torch.load with map_location=torch.device('cpu') to map your storages to the CPU.

**Comment by valentynbez** [comment]
I had the same error, it was fixed in #23.
Try to install the version from the repository - clone the repo and run `pip install .` in it.

**Comment by mortonjt** [comment]
Hi, thanks for catching this.  We just submitted a new pypi release https://pypi.org/project/tm-vec/#history

So the current pypi version should be up to date.
